### PR TITLE
Add TensorFlow eager execution example.

### PR DESCRIPTION
**Pull request by Rishav1**
Addresses #351

### CODE REVIEW

File: examples/tensorflow_eager_simple.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                optimizer.apply_gradients(zip(grads, model.variables))
+                if batch % LOG_INTERVAL == 0:
+                    print('Step #%d\tLoss: %.6f' % (batch, loss_value))
+
+    if mode == 'eval':
+        return avg_loss, accuracy
+
+
+def get_mnist():
+    # Load the data, split between train and test sets
```
Comment (sile) [inline examples/tensorflow_eager_simple.py]: Please append a trailing period.

**Comment by sile** [comment]
Could you shorten the training time by reducing the data set size? (a similar motivation to https://github.com/pfnet/optuna/pull/366#discussion_r273303672)

**Comment by Rishav1** [comment]
Sorry for not addressing the changes before, I have been busy with other commitments.

**Comment by sile** [comment]
Thank you!
Please merge `master` branch to fix CI failures (see also https://github.com/pfnet/optuna/pull/432).

**Comment by g-votte** [comment]
Thanks for your contribution!

**Pull request merged**